vaadin-time-picker-flow-parent/vaadin-time-picker-flow/src/main/resources/META-INF/resources/frontend/helpers.js

```diff
@@ -0,0 +1,123 @@
+// map from unicode eastern arabic number characters to arabic numbers
```

Comment:
When building the frontend for an application, Flow will collect all sources from all `resources/frontend` folders, and collect them in a single folder. That means that adding  a generic file like `helpers.js` could conflict with sources from other JARs or application code. I would suggest to move this into a component specific folder like `vaadin-time-picker`. See how the connector files for the Map component are structured for example.

Reply:
Oh, good that you spotted that, thanks. 👍  Moved both `helpers.js` and `timepickerConnector.js` to the `vaadin-time-picker` sub-folder.